seatunnel-engine/seatunnel-engine-storage/imap-storage-api/src/main/java/org/apache/seatunnel/engine/imap/storage/common/ProtoStuffSerializer.java

```diff
@@ -18,7 +18,7 @@
  *
  */
 
-package org.apache.seatunnel.engine.imap.storage.api.common;
```

Comment:
Why change this package name, it is a storage api thing, I don't want to introduce any conflicts later.

Reply:
> Why change this package name, it is a storage api thing, I don't want to introduce any conflicts later.

I'm just trying to be consistent with checkpot-storage-api.

Reply:
This is what you should focus on, https://github.com/apache/incubator-seatunnel/pull/3792/files#diff-b324912bbe79a7d26d9a89e70c92dc73b2337e070f193e65e8f8fb6bd803790bL41 not the so-called changing package name, that doesn't make any sense

Reply:
![image](https://user-images.githubusercontent.com/16631152/209561607-42cfcc6e-17cb-4fa6-b901-5bdf756e0561.png)